src/main/kotlin/uk/gov/justice/digital/hmpps/pollpush/config/JmsConfig.kt

```diff
@@ -31,7 +31,7 @@ open class JmsConfig {
     val factory = DefaultJmsListenerContainerFactory()
     factory.setConnectionFactory(SQSConnectionFactory(ProviderConfiguration(), awsSqs))
     factory.setDestinationResolver(DynamicDestinationResolver())
-    factory.setConcurrency("3-10")
+    factory.setConcurrency("1")
```

Comment:
don't want to overwhelm delius to start with

Reply:
Do you think essentially this means pull one - push one e.g single threaded? I suppose we could compare this with akka threads (?) in old poll pusher. Certainly a good starting point.

Reply:
well yes, however we'll be running 4 servers in live